### PR TITLE
missed a spot

### PR DESCRIPTION
**Pull request by kraftpunk97-zz**

### CODE REVIEW

**Comment by jbrea** [comment]
Nice. How did you spot it? Can you maybe add a little test such that we wont miss it next time? (I can also do it if you don't have time).

**Comment by kraftpunk97-zz** [comment]
Got it. Will add more commits to this PR with included tests.

**Comment by kraftpunk97-zz** [comment]
@jbrea @findmyway please review

File: test/runtests.jl [hunk 1]
```diff
@@ -40,15 +40,32 @@ end
...
+        restoreState(ale, state_ref)
+        deleteState(state_ref)
+        state_ref = cloneSystemState(ale)
+        state = encodeState(state_ref)
+        state_ref = decodeState(state)
+        restoreSystemState(ale, state_ref)
+        deleteState(state_ref)
+
+
         @test true
```
Comment (findmyway) [inline test/runtests.jl]: How about changing `@test true` to some more meaningful checks here. Like comparing `original_encoded_state` and `restored_encoded_state`?

File: src/aleinterface.jl [hunk 2]
```diff
@@ -436,7 +436,7 @@ saveScreenPNG(ale::ALEPtr, filename::String) = ccall((:saveScreenPNG, libale_c),
 
 function encodeState(state::ALEStatePtr)
```
Comment (findmyway) [inline src/aleinterface.jl:437]: We can also add an in-place method `encodeState!` to avoid allocating every time.

Comment (kraftpunk97-zz) [reply src/aleinterface.jl:437]: Are you suggesting including `encodeState` into the `cloneSystemState` and `cloneState`methods?

File: test/runtests.jl (hunk 1 above)
Comment (kraftpunk97-zz) [reply test/runtests.jl]: @findmyway Made the requested changes. Please review.

File: src/aleinterface.jl (hunk 2 above)
Comment (findmyway) [reply src/aleinterface.jl:437]: I've made a patch 😄

**Pull request merged**